docs/source/install/index.rst

```diff
@@ -54,6 +54,7 @@ Once it completes successfully, you will see the following output:
     Ubuntu 14.04 / 16.04 <deb>
     RHEL 7 / CentOS 7 <rhel7>
     RHEL 6 / CentOS 6 <rhel6>
+    Ansible Playbooks <ansible>
```

Comment:
Alternative approach is to polish and revert entire 'puppet_chef_salt_ansible.rst' page what possibly can encourage contributions for other installation methods. In a meantime I left that page if we reconsider in some future time. WDYT?